packages/seller-client/src/pages/ProductsPage/ProductsPage.tsx

```diff
@@ -36,6 +36,7 @@ const ProductsPage: React.FC = () => {
           <ProductListItem
             key={product._id}
             {...product}
+            image={product.images?.[0]}
```

Comment:
I think `product.images` will always be defined as an array so no need for the optional chaining (`?.`)

if the array is empty it will return undefined anyways for `product.images[0]`
```suggestion
            image={product.images[0]}
```

Reply:
Ah you're right. Updated in 43a0227ca4e4f148467cc9b61425f4da9a4922d9